projects/archiver/src/notifications/device-notifications.ts

```diff
@@ -1,30 +1,66 @@
-import { IssuePublicationIdentifier, Issue } from '../../common'
+import uuid from 'uuidv4'
+import { attempt } from '../../../backend/utils/try'
+import fetch from 'node-fetch'
+import dateFormat from 'dateformat'
 
-interface Topic {
-    type: string
+type EditionNotificationTypes = 'editions'
+
+type NotificationSenders = 'editions-backend'
+
+interface EditionTopic {
+    type: EditionNotificationTypes
     name: string
 }
 
-interface ScheduleAppStoreNotificationPayload {
+interface ScheduleDeviceNotificationPayload {
     id: string
-    type: string
-    topic: Topic[]
+    type: EditionNotificationTypes
+    topic: EditionTopic[]
     key: string
     name: string
     date: string
-    sender: string
+    sender: NotificationSenders
+}
+
+/**
+ * TODO
+ * it will work now noly for Daily Editions in UK
+ * In the future we will need to make it more generic (for US and Australia)
+ **/
+const createDatePathParam = (now: Date) => {
+    const THREE_AT_NIGHT = 'T03:00:00Z'
+    return `${dateFormat(now, 'yyyy-mm-dd')}${THREE_AT_NIGHT}`
```

Comment:
Wont this make the return be in the past if this function runs after 3am on the current day?

Reply:
yes, so i think we may want to ad + one day, as for today test the schedule was posted for `2019-09-30T03:00:00Z`

Reply:
good spot thanks for that!!!

Reply:
i changed that to be always tomorrow at 3 am from trigger time

Reply:
we decided to use issue date instead